src/frontend/Komponenter/Personoversikt/utils.ts

```diff
@@ -27,3 +28,19 @@ export function kanOppretteRevurdering(fagsak: Fagsak) {
         harBehandlingMedTypeFørstegangsbehandlingEllerRevurdering
     );
 }
+
+export const oppdaterVedleggFilter = (
+    object: VedleggRequest,
+    key: keyof VedleggRequest,
+    val?: string | number
+): VedleggRequest => {
+    if (!val || val === '') {
+        // eslint-disable-next-line
+        const { [key]: dummy, ...remainder } = object;
```

Comment:
Skal det stå dummy her?

Reply:
Siden den brukes til å sette feltet til null i stedet for en tom streng, så er kanskje `setNull` et bedre navn?